src/scaict_uwu/service_wiring.py

```
@@ -0,0 +1,40 @@
+"""
+Module for SCAICT-uwu default service implementations.
+"""
+
+# Standard imports
+from typing import Callable
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)